src/Map.jsx

```diff
@@ -130,6 +132,19 @@ class Map extends Component {
   /**
    * Calculate & Update state of new dimensions
    */
+
+  setNewCity(clicked) {
+    let newCity = {};
+    cities.map((city) => { if (city.name === clicked.value) newCity = city; return null; });
```

Comment:
You don't need to `return null` here; if you leave nothing then you're implicitly returning `undefined`.
Also, we could use `forEach` instead of `map` here as we're not using the result.